### DISCUSSION TITLE
Fix tabs when JS is off

### DISCUSSION THREAD

**Discussion by LucyGwilliamAdmin** [open-sdg/open-sdg#1610] [Ideas]
When JS is off, metadata tab section is not user friendly - all metadata sections display under national metadata tab but this isn't obvious. Additionally the other tabs display, making users think they can be clicked

**To discuss: how can we resolve this? when JS is off should the design that has been suggested for mobile be implemented?**

<img src="https://user-images.githubusercontent.com/43404358/168615904-ededae55-9072-4bcb-8783-eca66bbf2e72.png" width="35%" height="35%">

Metadata section with JS off:
<img src="https://user-images.githubusercontent.com/43404358/168617131-7427f224-51ad-4c8f-9254-50c7527580e9.png" width="75%" height="75%">
<img src="https://user-images.githubusercontent.com/43404358/168617459-1f39f505-33ed-4cd9-ae2c-d56f3d6e8e9f.png" width="75%" height="75%">

**Comment by LucyGwilliamAdmin** [comment]
@phillipgardiner @brockfanning @otis-bath thoughts please!

**Reply by otis-bath** [reply]
@LucyGwilliamAdmin Feel like the mobile menu style you have shown would be fine, as long as there are clear headers and the tabs aren't present this would be clear what is being referred to. Not sure how easy this would be to implement though on all these pages?

**Reply by phillipgardiner** [reply]
@LucyGwilliamAdmin @otis-bath @LucyGwilliamAdmin If we are able to start with the design proposed as the baseline on which to build up then that would be great.

**Reply by LucyGwilliamAdmin** [reply]
@brockfanning how difficult would it be to implement the proposed design for when JS is off?

**Reply by brockfanning** [reply]
@LucyGwilliamAdmin Sorry, not sure how I missed this thread. I will look into this.

**Comment by LucyGwilliamAdmin** [comment]
Note: same thing occurs on reporting status page i.e. both Reporting status and Disaggregation status tabs show but Disaggregation status shows below reporting status and Disaggregation status tab is not clickable
